components/crowdfunding-redesign/edit/common.tsx

```diff
@@ -0,0 +1,236 @@
+import { Label } from '../../ui/Label';
+import { Popover, PopoverContent, PopoverTrigger } from '../../ui/Popover';
+
+export function FormField({
+  label,
+  showError = true,
+  name,
+  hint,
+  placeholder,
+  children,
+  ...props
+}: {
+  label?: string;
+  showError?: boolean;
+  name: string;
+  hint?: string;
+  placeholder?: string;
+  children?: (props: { form: FormikProps<any>; meta: any; field: any }) => JSX.Element;
+  required?: boolean;
+  min?: number;
+  max?: number;
+  inputType?: string;
+  disabled?: boolean;
+  htmlFor?: string;
+}) {
+  const intl = useIntl();
+  const htmlFor = props.htmlFor || `input-${name}`;
+  const { schema } = useContext(FormikZodContext);
+
+  return (
+    <Field name={name}>
+      {({ field, form, meta }) => {
+        const hasError = Boolean(meta.error && (meta.touched || form.submitCount));
+        const fieldAttributes = {
+          ...(schema ? getInputAttributesFromZodSchema(schema, name) : null),
+          ...pickBy(
+            {
+              ...field,
+              name: name || htmlFor,
+              id: htmlFor,
+              type: props.inputType,
+              disabled: props.disabled,
+              min: props.min,
+              max: props.max,
+              required: props.required,
+              error: hasError,
+              placeholder,
+            },
+            value => value !== undefined,
+          ),
+        };
+        if (
+          'required' in fieldAttributes &&
+          !fieldAttributes.required &&
+          meta.error &&
+          meta.error === intl.formatMessage(RICH_ERROR_MESSAGES.requiredValue)
+        ) {
+          fieldAttributes.required = true;
+        }
+        return (
+          <div className="flex w-full flex-col gap-1">
+            {label && <Label className="text-base leading-normal">{label}</Label>}
+            {hint && <p className="text-sm text-muted-foreground">{hint}</p>}
+            {children ? children({ form, meta, field: fieldAttributes }) : <Input {...fieldAttributes} />}
+            {hasError && showError && (
+              <p className="text-xs text-red-500">
+                {isOCError(meta.error) ? formatFormErrorMessage(intl, meta.error) : meta.error}
+              </p>
+            )}
+          </div>
+        );
+      }}
+    </Field>
+  );
+}
+
+export function ColumnSection({ title, description, children }) {
+  return (
+    <div className="grid grid-cols-12 gap-4">
+      <div className="col-span-full md:col-span-4">
+        <div className="text-lg font-medium">{title}</div>
+        <p className="text-sm text-muted-foreground">{description}</p>
+      </div>
+      <div className="col-span-full md:col-span-8 md:col-start-6">
+        <div className="flex flex-col gap-4">{children}</div>
+      </div>
+    </div>
+  );
+}
+
+function ColorPicker({ value, onChange }) {
+  // Selected colors from tailwind palette
+  const options = [
+    '#ef4444', // red-500
```

Comment:
I don't think we should suggest red as a primary color since it conveys a sense of danger, and having a donate button that looks dangerous is probably a bad idea :-) In the previous color picker, we tried to avoid that by tending toward orange/brown rather than red - and IMO some of them still look too reddish.

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/f08b95e3-4814-49ac-a6a9-c1607af1553a)

People can still put that manually if they want to, but we should ideally suggest "best practices".

Reply:
Good idea. I've removed the red option entirely.